src/mame/drivers/mc10.cpp

```diff
@@ -247,26 +257,30 @@ DRIVER_INIT_MEMBER(mc10_state,mc10)
 	/* initialize memory */
 	m_ram_base = m_ram->pointer();
 	m_ram_size = m_ram->size();
-	m_pr_state = PRINTER_WAIT;
+	m_pr_state = (uint8_t)printer_state::WAIT;
 
-	membank("bank1")->set_base(m_ram_base);
+	m_bank1->set_base(m_ram_base);
 
 	/* initialize memory expansion */
 	if (m_ram_size == 20*1024)
-		membank("bank2")->set_base(m_ram_base + 0x1000);
+		m_bank2->set_base(m_ram_base + 0x1000);
 	else if (m_ram_size == 24*1024)
-		membank("bank2")->set_base(m_ram_base + 0x2000);
+		m_bank2->set_base(m_ram_base + 0x2000);
 	else  if (m_ram_size != 32*1024)        //ensure that is not alice90
 		prg.nop_readwrite(0x5000, 0x8fff);
 
 	/* register for state saving */
 	save_item(NAME(m_keyboard_strobe));
+	save_item(NAME(m_pr_state));
+	save_item(NAME(m_pr_counter));
 
 	//for alice32 force port4 DDR to 0xff at startup
 	if (!strcmp(machine().system().name, "alice32") || !strcmp(machine().system().name, "alice90"))
 		m_maincpu->m6801_io_w(prg, 0x05, 0xff);
 }
 
+}
```

Comment:
Why not put the address maps etc. inside the anonymous namespace so you can ditch the `static` keyword everywhere?